Dockerfile

```diff
@@ -7,6 +7,12 @@ ENV PATH /usr/local/rbenv/bin:$PATH
 
 RUN npm install gulp -g
 
+VOLUME /work
+WORKDIR /work
+
+COPY omega_4_4/* /work/omega_4_4/
```

Comment:
I'd better use different folder for gemsets to prevent collisions when /work dir mounted via docker run